e2e/test/dom.spec.ts

```diff
@@ -6,6 +6,13 @@ describe('DOM', () => {
   let screen: WebdriverIOQueries;
 
   before(() => {
+    /**
+     * This is a workaround for the issue with the `browser` object type being
+     * mismatched`.
+     * @see https://github.com/testing-library/webdriverio-testing-library/issues/51
```

Comment:
These are fairly annoying, seems `webdriverio-testing-library` is overdue a bit of love.